src/Corax/Querying/IndexSearcher.cs

```diff
@@ -48,7 +48,7 @@ public sealed unsafe partial class IndexSearcher : IDisposable
     /// When true no SIMD instruction will be used. Useful for checking that optimized algorithms behave in the same
     /// way than reference algorithms. 
     /// </summary>
-    public bool ForceNonAccelerated { get; set; }
+    public bool ForceNonAccelerated { get; set; } = true;
```

Comment:
That should not be committed, should it?
Also, do we have a test to verify that we aren't accidently leaving it like that?

Reply:
1) It shouldn't be, it's part of TEMP commit to run all tests with scalar code.
2) No, I will add a test for this.